include/iota/crypto/signing.hpp

```diff
@@ -28,34 +28,27 @@
 #include <iostream>
 #include <vector>
 
-#include <iota/crypto/kerl.hpp>
+#include <iota/types/trinary.hpp>
 
 namespace IOTA {
 
 namespace Crypto {
 
-class Signing {
-public:
-  Signing();
-  virtual ~Signing();
-
-public:
-  Types::Trits key(const Types::Trytes& seed, const unsigned int& index = 0,
-                  const unsigned int& security = 1);
-  Types::Trits digest(const std::vector<int8_t>& normalizedBundleFragment,
-                     const Types::Trits&         signatureFragment);
-  Types::Trits digests(const Types::Trits& key);
-  Types::Trits address(const Types::Trits& digests);
-  Types::Trits signatureFragment(const std::vector<int8_t>& normalizedBundleFragment,
-                                const Types::Trits&         keyFragment);
-  bool        validateSignatures(const Types::Trytes&              expectedAddress,
-                                 const std::vector<Types::Trytes>& signatureFragments,
-                                 const Types::Trytes&              bundleHash);
-
-private:
-  // TODO Sponge
-  Kerl kerl_;
-};
+namespace Signing {
+
+Types::Trits key(const Types::Trytes& seed, const unsigned int& index = 0,
```

Comment:
maybe same stuff as previous PR, if you can include additional doc for these ones, would be perfect

Reply:
There are specific issues for that, but sure, I'll close them all in this PR later today.

Reply:
ok no worry, I'll merge this one and just do the other specific issue when you want